.github/workflows/newbuild-vcpkg.yml

```diff
@@ -0,0 +1,333 @@
+name: build-vcpkg
+
+env:
+  VCPKG_BINARY_SOURCES: "clear;nuget,GitHub,read"
+  VCPKG_NUGET_REPOSITORY: "https://github.com/hpcc-systems/vcpkg"
+
+on:
+  push:
+    tags:
+      - "newbuild_*"
```

Comment:
question: is "newbuild_" just for testing? If not, when does a branch have a tag like this?

Reply:
Just for testing

Reply:
right. Maybe add a comment if really necessary to keep in master branch.
Is it not possible to do same testing on your repo with the standard branch names?